assessment.sql

```diff
@@ -5,24 +5,53 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
-
+Create table licensees(
+  id serial primary key,
+  surname character varying,
+  given_name character varying,
+  gender character varying(1),
+  height integer,
+  weight, integer,
```

Comment:
That comma between `weight` and `integer` will cause Postgres to complain that you did not specify a type for the column.